src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (x) => {
+    if (x === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
+
+    sum += x;
```

Comment:
It's a good practice to ensure that 'x' is a number before adding it to 'sum'. You can use 'Number.isFinite(x)' to check if 'x' is a finite number.